collector/settings.mdx

```diff
@@ -174,6 +174,11 @@ helper functions are expected to be defined, and the one we connect to for serve
       <td>[none]</td>
       <td>Alternative to above, using actual contents of the key</td>
     </tr>
```

Comment:
I wonder if this wouldn't be a better fit for the "Self-managed servers" section, since its only relevant in that case, and `db_log_location` also lives there.